modules/programs/khard.nix

```
@@ -68,6 +68,11 @@ in {
       type = with lib.types;
         attrsOf (submodule {
           options.khard.enable = lib.mkEnableOption "khard access";
+          options.khard.defaultCollection = lib.mkOption {
+            type = types.str;
+            default = "";
+            description = "Sub-vdir to hardcode for khard";
```

I don't quite understand this, but I think this is more descriptive of what it is.

```suggestion
            description = "Sub-vdir where the khard address book resides.";
```

---

Thank you very much for the review!
I accidentally destroyed the src branch for the PR, but I've already took these into account, could we continue on #5220?